Test/comp/BranchCoverage.py

```diff
@@ -0,0 +1,22 @@
+import sys
+
+assert "DafnyProfiling" == __name__
```

Comment:
Shouldn't this say `BranchCoverage`?

Reply:
This is hard coded in the SinglePassCompiler: https://github.com/dafny-lang/dafny/blob/65ae35bfc0ab237975b6a2594c55d67adf2a8eee/Source/Dafny/Compilers/SinglePassCompiler.cs#L5357

Reply:
But then shouldn't the file be called `DafnyProfiling.py`?

Reply:
Oh, I see. We rename the file according to the assert. This gives use more flexibility.